ios/RCCNavigationController.m

```diff
@@ -326,8 +327,20 @@ -(void)setButtons:(NSArray*)buttons viewController:(UIViewController*)viewContro
     id icon = button[@"icon"];
     if (icon) iconImage = [RCTConvert UIImage:icon];
     
+    id systemIcon =button[@"buttonSystemItem"];
+    UIBarButtonSystemItem systemItemIcon=NSNotFound;
+    if (systemIcon) systemItemIcon =[RCTConvert UIBarButtonSystemItem:button[@"buttonSystemItem"]];
+
+    id buttonColorTag = button[@"buttonColor"];
+    UIColor *buttonColor = nil;
+    if (buttonColorTag) buttonColor= [RCTConvert UIColor:buttonColorTag];
+
+
     UIBarButtonItem *barButtonItem;
-    if (iconImage)
+    if (systemItemIcon!=NSNotFound) {
```

Comment:
Nice catch in noticing that [`-[RCTConvert UIBarButtonSystemItem:]` returns `NSNotFound` by default](https://github.com/facebook/react-native/blob/40fdd6d91c578934b9e1035a167888032fb85d28/React/Views/RCTNavItemManager.m#L42)!